### ISSUE TITLE
Export as markdown doesn’t include subnotebook folders

### ISSUE THREAD

**Issue by cgreenberg** [HappenApps/Quiver#1223]
Not sure if this is a bug or a feature, but when I export a notebook as markdown it doesn’t include the sub folder structure, instead all notes are dumped as one flat list of md files. Would be helpful to export and make folders of subnotebooks.

**Comment by ylian** [comment]
Good idea. The flat list is only because exports were implemented way before nested notebooks are supported. I will fix this.

**Comment by l0rda** [comment]
Hi,

any news on this?

**Comment by ErrorSource** [comment]
Hi, would be interessted too. Any possibilty to implement this?
